pkg/apis/recognizer/devfile_recognizer.go

```diff
@@ -268,6 +264,12 @@ var DownloadDevfileTypesFromRegistry = func(url string, filter model.DevfileFilt
 	return devfileTypes, nil
 }
 
+func closeHttpResponseBody(resp *http.Response){
```

Comment:
@Jdubrick 

I'd suggest renaming this to `CloseHttpResponseBody` and place it inside the `detector.go` (utils package). This way it can be imported both in `recognizer` & `main` (check_registry) and we can have the implementation only once.

Initialy I thought we couldn't do that but it was okay when I tested it WDYT?

Reply:
I will make the change!